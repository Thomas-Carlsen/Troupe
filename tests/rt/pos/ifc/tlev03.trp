let val x = 42 raisedTo {alice}
    val y = 11 raisedTo {bob} 
    val z = (x,y)
    val (a,b) = z      
in a
end
