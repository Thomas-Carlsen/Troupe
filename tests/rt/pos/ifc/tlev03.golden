2019-03-07T18:56:06.229Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: 42@{alice}%{}
